liqd_product/templates/base.html

```diff
@@ -36,6 +36,7 @@
     <a href="#main" class="sr-only sr-only-focusable">{% trans "Skip to content "%}</a>
 
     {% include 'header.html' %}
+    {% block after_header %}{% endblock %}
```

Comment:
What is this useful for? Why is this not directly implemented in the header itself? (Same for `before_footer`)